cgi/test_ingredients_analysis.pl

```diff
@@ -44,30 +44,21 @@
 
 ProductOpener::Display::init();
 
+my $template_data_ref = {
+	lang => \&lang,
```

Comment:
There is a new process_template() function that adds lang => \&lang by default, so it's not needed here.

Reply:
Alright, let me fix it.